### PR TITLE
LPS-143330 [REACT] Update Card component with Dialect theme

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

**Comment by silasbsi** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Basics/BusinessType/Search.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const MAX_LENGTH_TO_TRUNCATE = 28;
 
 export function BusinessTypeSearch({form, setNewSelectedProduct}) {
 	const {
-		formState: {errors},
+		// formState: {errors},
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Basics/BusinessType/Search.js:27]: this comment could be removed

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Basics/BusinessType/Search.js [hunk 2]
```diff
@@ -122,7 +123,11 @@ export function BusinessTypeSearch({form, setNewSelectedProduct}) {
 		}
 
 		if (isError) {
-			return <WarningBadge>{isError}</WarningBadge>;
+			return (
+				<>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Basics/BusinessType/Search.js:127]: Remove this fragment 

```
<>
	<WarningBadge>{isError}</WarningBadge>
</>
```

and let only: ```					<WarningBadge>{isError}</WarningBadge>```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Hey @ilzamcmed it's looks pretty good!!

I'm really happy with this PR :D 

I left just few adjusts to do.

About the commit message, [REACT] is not necessary in the commit message, can you remove?

**Comment by eduardobbgf** [comment]
Just starting reviewing :)

**Comment by eduardobbgf** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**